mmvb_backend/case_synthesizer/generator.py

```diff
@@ -48,11 +49,15 @@ def sample_symptoms(symptom_probabilities):
 
 
 def combine_symptom_and_state(symptom, state_value):
-    # Combining two dictionaries together:
+    """Combines two dictionaries together"""
```

Comment:
Isn't this too general? I updates the symptom dictionary with the values from the state_value?